libs/ui/elements/src/lib/metadata-contact/metadata-contact.component.html

```diff
@@ -1,32 +1,94 @@
+        (click)="onOrganizationClick()"
+        data-cy="organization-name"
+      >
+        {{ shownOrganization.name }}
+      </div>
+    </div>
+    <div *ngIf="shownOrganization.website" class="mb-6">
+      <a
+        [href]="shownOrganization.website"
+        target="_blank"
+        class="contact-website text-primary text-sm cursor-pointer hover:underline transition-all"
+        >{{ shownOrganization.website }}
+        <mat-icon
+          class="material-symbols-outlined !w-[12px] !h-[12px] !text-[12px] opacity-75 shrink-0"
+          >open_in_new</mat-icon
+        >
+      </a>
+    </div>
+  </div>
+  <div *ngIf="contacts[0].phone" class="mb-5">
+    <div class="flex">
+      <mat-icon
+        class="material-symbols-outlined !w-5 !h-5 !text-[20px] opacity-75 shrink-0"
+        >call_outline</mat-icon
+      >
+      <div class="flex flex-col ml-2">
+        <p class="text-sm">{{ contacts[0].phone }}</p>
+      </div>
     </div>
   </div>
-  <a
-    *ngIf="contacts.length"
-    [href]="'mailto:' + contacts[0].email"
-    class="text-gray-700 text-sm hover:underline"
-    target="_blank"
-    >{{ contacts[0].email }}</a
-  >
-  <div *ngIf="shownOrganization.website" class="mb-2">
-    <a
-      [href]="shownOrganization.website"
-      target="_blank"
-      class="contact-website text-primary text-sm cursor-pointer hover:underline transition-all"
-      >{{ shownOrganization.website }}
+  <div *ngIf="contacts[0].email" class="mb-5">
+    <div class="flex">
       <mat-icon
-        class="material-symbols-outlined !w-[12px] !h-[12px] !text-[12px] opacity-75"
-        >open_in_new</mat-icon
+        class="material-symbols-outlined !w-5 !h-5 !text-[20px] opacity-75 shrink-0"
+      >
+        mail_outline</mat-icon
       >
-    </a>
+      <a
+        *ngIf="contacts.length"
+        [href]="'mailto:' + contacts[0].email"
+        class="text-sm hover:underline ml-2"
+        target="_blank"
+        data-cy="contact-email"
+        >{{ contacts[0].email }}</a
+      >
+    </div>
+  </div>
+  <div *ngIf="contacts[0].firstName || contacts[0].lastName" class="mb-5">
+    <div class="flex">
+      <mat-icon
+        class="material-symbols-outlined !w-5 !h-5 !text-[20px] opacity-75 shrink-0"
+        >person_outline</mat-icon
+      >
+      <div class="flex flex-col ml-2">
+        <p class="text-sm">
+          {{ contacts[0].firstName }} {{ contacts[0].lastName }}
+        </p>
+      </div>
+    </div>
+  </div>
+  <div *ngIf="contacts[0].address" class="mb-2">
+    <div class="flex">
+      <mat-icon
+        class="material-symbols-outlined !w-5 !h-5 !text-[20px] opacity-75 shrink-0"
+      >
+        location_on</mat-icon
+      >
+      <div class="flex flex-col ml-2">
+        <p class="text-sm">
+          {{ address[0] }}
+        </p>
+        <p *ngFor="let addressPart of address.slice(1)" class="text-sm">
+          {{ addressPart }}
+        </p>
```

Comment:
why would we need to split the logic here? having `ngFor` loop on all parts would be enough I guess